draft-ietf-jsonpath-base.md

```diff
@@ -569,207 +583,257 @@ of node.
-| "\\" "/"           | U+002F              |
-| "\\" "\\"          | U+005C              |
-| "\\" "b"           | U+0008              |
-| "\\" "f"           | U+000C              |
-| "\\" "n"           | U+000A              |
-| "\\" "r"           | U+000D              |
-| "\\" "t"           | U+0009              |
-| "\\" uXXXX         | U+XXXX              |
+| Escape Sequence    | Unicode Character   |  Description                |
+| :----------------: | :-----------------: |:--------------------------- |
+| \\b                | U+0008              | BS backspace                |
+| \\t                | U+0009              | HT horizontal tab           |
+| \\n                | U+000A              | LF line feed                |
+| \\f                | U+000C              | FF form feed                |
+| \\r                | U+000D              | CR carriage return          |
+| \\"                | U+0022              | quotation mark              |
+| \\'                | U+0027              | apostrophe                  |
+| \\/                | U+002F              | slash (solidus)             |
+| \\\\               | U+005C              | backslash (reverse solidus) |
+| \\uXXXX            | U+XXXX              | unicode character           |
 {: title="Escape Sequence Replacements" cols="c c"}
 
-The child selects the value corresponding to the member name from any object
-node that has a member with that name.
-It selects no nodes from a node which is not a object.
+The `index-selector` applied with a `quoted-member-name` to an object
+selects the node of the corresponding member value from it, if and only if that object has a member with that name.
+Nothing is selected from a value which is not a object.
 
+Array indexing via `element-index` is a way of selecting a particular array element using a zero-based index.
+For example, selector `[0]` selects the first and selector `[4]` the fifth element of a sufficiently filled array.
 
+A negative `element-index` counts from the array end.
+For example, selector `[-1]` selects the last and selector `[-2]` selects the last but one element of an array with at least two elements.
 
-#### Array Selector
 
-##### Syntax
-{: numbered="false" toc="exclude"}
+### Index Wild Card Selector
 
-An array selector selects zero or more elements of an array node.
-An array selector takes the form of an index, which selects at most one element,
-or a slice, which selects zero or more elements.
+#### Syntax
+{: unnumbered}
+
+The index wild card selector has the form `[*]`.
 
 ~~~~ abnf
-union-element =/ array-index / array-slice
+index-wild-selector    = "[" "*" "]"  ;  asterisk enclosed by brackets
 ~~~~
 
-An array index is an integer (in base 10).
+#### Semantics
+{: unnumbered}
 
-~~~~ abnf
-array-index = integer
+An index selector combined with a single asterisk is a wild card. It
+selects all member values of an object as well as all elements of an
+array.
```

Comment:
Should talk about selecting nodes rather than values.

Reply:
Fixed.